create.sql

```diff
@@ -772,7 +797,7 @@ COPY e_template (id, short_name, long_name, tcc_value, obsolete) FROM stdin;
 --
 
 COPY gem_user (id, first, last, md5, email, staff) FROM stdin;
-root			                                	 	t
+tpolecat	Rob	Norris	72b302bf297a228a75730123efef7c41	rnorris@gemini.edu	f
```

Comment:
Yeah the `root` user is required by the importer, which is why you got the error above. Unclear why it went away and mine showed up. I think maybe you need to rebase.